process/block/metablock_test.go

```diff
@@ -3338,7 +3346,11 @@ func TestMetaProcessor_CreateEpochStartBodyShouldWork(t *testing.T) {
 	t.Run("rewards V2 enabled", func(t *testing.T) {
 		t.Parallel()
 
-		arguments := createMockMetaArguments(coreComponents, dataComponents, bootstrapComponents, statusComponents)
+		cc := coreComponents
+		cc.EnableEpochsHandlerField = &testscommon.EnableEpochsHandlerStub{
+			IsScheduledMiniBlocksFlagEnabledField: true,
+		}
```

Comment:
is this flag required for rewards V2 enabled test?

Reply:
correct, not needed..also removed it from other places